js/viz.js

```diff
@@ -325,13 +351,11 @@ const viz = {
 
   dragEnd() {
     d3.event.subject.shadow = false;
-    this.endEvent();
```

Comment:
I would like it not to throw the whole graph around yeah, things can move but recalculating the whole graph shouldn't happen within one paint certainly.